### ISSUE TITLE
None in Priority and Severity dropdown should be removed in Agile template

### ISSUE THREAD

**Issue by Raunak1203** [openshiftio/openshift.io#3831]
- Create a work item of type Defect
- Click on the workItem title to open quickPrebiew
- Select Priority or Severity drop-down
- None is also displayed in the drop-down which should be removed

![screenshot from 2018-06-20 10-54-28](https://user-images.githubusercontent.com/34271052/41638808-dd39cca4-7478-11e8-81d6-906c02e8e0ca.png)

**Comment by joshuawilson** [comment]
@kwk is this value coming from the backend? If so, should it be removed there? 

@openshiftio/uxd-team is there ever an case where you might want to set it to `none`?

**Comment by kwk** [comment]
@joshuawilson. The backend should by now set the severity or any other enum typed field to the first value in the list of allowed enums or to the default value if given. See this file: https://github.com/fabric8-services/fabric8-wit/blob/master/spacetemplate/assets/agile.yaml

That being said, no the None value does not come from the backend. Can you please see if this is still an issue? Sudipta knows where None comes from in the list. But it should not be selectable and should not be the default. 



It’s a holiday in Germany today. Sorry that I cannot write more from my phone.

**Comment by joshuawilson** [comment]
It is preset to a severity but the `none` is still present. I just don't know if it should stay or not.

**Comment by kwk** [comment]
It should not stay.

**Comment by jarifibrahim** [comment]
@joshuawilson @kwk I agree the `None` shouldn't be in the list. I've raised https://github.com/fabric8-ui/fabric8-planner/pull/2798 to fix it.

**Comment by jarifibrahim** [comment]
I'll close the issue once the changes are in prod.
